composables/transaction/mintCollection/constructMeta.ts

```diff
@@ -0,0 +1,57 @@
+import { ActionMintCollection } from '../types'
+import { createMetadata, unSanitizeIpfsUrl } from '@kodadot1/minimark'
+import { pinJson } from '@/services/nftStorage'
+import { usePinningStore } from '@/stores/pinning'
+import { uploadDirect } from '@/utils/directUpload'
+import { getImageTypeSafe, pinImageSafe } from '@/utils/safePin'
+import { Attribute } from '@kodadot1/minimark'
+
+const createAttributes = ({
+  tags,
+  urlPrefix,
+}: {
+  tags: Attribute[]
+  urlPrefix: string
+}) => {
+  if (urlPrefix === 'bsx' || urlPrefix === 'snek') {
+    return tags.map((val) => ({
+      ...val,
+      display_type: null,
+    }))
+  }
+  if (urlPrefix === 'rmrk' || urlPrefix === 'rmrk2') {
+    return []
+  }
```

Comment:
why so? tags should be also available in rmrk

Reply:
see line 166 in /rmrk/create/createCollection (current code, in main)

createMetaData for RMRK collection was passed empty array for attributes